composer.json

```diff
@@ -411,7 +411,7 @@
         "simplesamlphp/simplesamlphp-module-openidprovider": "<0.9",
         "simplito/elliptic-php": "<1.0.6",
         "sitegeist/fluid-components": "<3.5",
-        "slim/psr7": "<1.6.1",
+        "slim/psr7": "<1.4.1|<1.5.1|<1.6.1",
```

Comment:
BTW, this range is insufficient: this still is equivalent to just `<1.6.1`.

Reply:
Yeah, it the right one is https://github.com/github/advisory-database/pull/2233#discussion_r1189601026